content/en/docs/contribute/new-content/overview.md

```diff
@@ -8,7 +8,7 @@ weight: 5
 
 {{% capture overview %}}
 
-This section contains information all contributors should know before beginning.
+This section contains information you should know before contributing new content.
```

Comment:
Line 20: Markdown
Might be nice to list the docs repository before commenting on the doc format and tool to build.
Line 21: nit: The source is in GitHub at ... seems redundant.
The Kubernetes documentation source is located at ... OR
You can find the Kubernetes documentation at ...
Line 23: capitalize `Custom Hugo shortcodes` vs custom Hugo shortcodes
Line 57: Capitalize, English?
Line 65: nit: Should `Base Branch` use capitalization?

Reply:
Thank you for these nits ;)